README.md

```diff
@@ -600,11 +618,14 @@ const res = ResultAsync.fromPromise(insertIntoDb(myUser), () => new Error('Datab
 
 Same as `ResultAsync.fromPromise` except that it does not handle the rejection of the promise. **Ensure you know what you're doing, otherwise a thrown exception within this promise will cause ResultAsync to reject, instead of resolve to a Result.**
 
-
 **Signature:**
 
 ```typescript
-fromSafePromise<T, E>(p: Promise<T>):  ResultAsync<T, E> { ... }
+class ResultAsync<T, E> {
+  fromSafePromise<T, E>(
+    promise: Promise<T>
+  ): ResultAsync<T, E> { ... }
+}
```

Comment:
Same comment as above regarding `fromSafePromise` being a static class method ... there is no prior `T` and `E` that you're operating on in this case.

Also, you have conflicting `T` and `E` values (e.g. you're redeclaring `T` and `E` on line 625).